# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ipw3945/depmod.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ipw3945-1.2.1/Makefile.vanilla	2007-06-14 18:34:07.000000000 +0000
+++ ipw3945-1.2.1/Makefile	2007-06-14 18:34:10.000000000 +0000
@@ -270,7 +270,7 @@
 install:
 	install -d $(KMISC)
 	install -m 644 -c $(addsuffix .ko,$(list-m)) $(KMISC)
-	/sbin/depmod -a ${KVER}
+	#/sbin/depmod -a ${KVER}
 	@echo "Don't forget to copy firmware to your hotplug's firmware directory "
 	@echo "and have the hotplug tools in place."
 	@echo "See INSTALL for more information."
